api_v3/services/ThumbAssetService.php

```diff
@@ -949,6 +952,21 @@ public function addFromImageAction($entryId, $fileData)
 		return $thumbAssets;
 	}
 	
+	protected function isValidFileType($fileName, $partnerId = null)
```

Comment:
since the function is now in thumb asset service lets give it a relevant name like isThumbValidFileType